starport/services/chain/chain.go

```diff
@@ -94,11 +95,31 @@ func New(app App, noCheck bool, logLevel LogLevel) (*Chain, error) {
 	if err != nil {
 		return nil, err
 	}
-	c.cmd = chaincmd.New(
-		app.D(),
+
+	ccoptions := []chaincmd.Option{
 		chaincmd.WithChainID(id),
 		chaincmd.WithHome(c.Home()),
-	)
+		chaincmd.WithKeyringBackend(chaincmd.KeyringBackendTest),
```

Comment:
Doing this we assume that all chains will use the `test` keyring backend. Currently, this is not what we have we chain used through `networkbuilder` for example

Reply:
Hm, I just checked the source code of `networkbuilder`, are you sure that we don't use test keyring backend there? Because it seems we don't specify this while adding accounts etc., and SDK chains by default uses the test backend.

PS, as a feature, I think we should let this to be configurable by the user for the `network` commands.

Reply:
I think the `os` keyring backend is used by default.
This works on Gitpod so it's ok for me to let it, the value will become configurable in https://github.com/tendermint/starport/pull/615

Reply:
Hm, yes, that's right. Only, the apps scaffold by Starport have it pointed to test. Regardless, we should be explicitly setting a value either as _test_ or _os_ to introduce a fixed behavior. @fadeev what're your thoughts on witch value we should set by default for `starport network` chains?

Reply:
I think `starport network` chains should use `os` keyring by default, because these commands are meant to run in "production" environments.

Reply:
tracked by https://github.com/tendermint/starport/issues/617.